### DISCUSSION TITLE
notifications not marked read after viewing them

### DISCUSSION THREAD

**Discussion by farmboy0** [community/community#20441] [Projects and Issues]
My notifications are not set as read after I view them and go back(with mouse button 4) to the notifications page. This includes releases, pull requests, and issues. This seems to happen randomly, not always.
Also the notification area switches back to all notification even if i switched to unread before.

**Comment by radar-blip** [comment]
If it only happens randomly, probably a bug.
You can file a bug report if you wish, here: https://support.github.com/contact/bug-report

**Reply by farmboy0** [reply]
Thanks, thats the url i was looking for

**Reply by radar-blip** [reply]
Anytime.
